include/common/mpmc_queue.h

```diff
@@ -40,7 +40,7 @@ class mpmc_queue{
    public:
       using value_type = T;
 
-      mpmc_queue<T>(int _size, queue_mode active = queue_mode::blocking ):
+      mpmc_queue<T>(int _size, queue_mode active):
```

Comment:
This argument have been renamed to "q_mode". Aditionally, the "_size" argument have been renamed to "q_size".